src/tsconfig.json

```diff
@@ -1,12 +1,16 @@
 {
     "compilerOptions": {
+        "target": "es5",
         "noImplicitAny": true,
         "noImplicitReturns": true,
         "noImplicitThis": false,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
         "suppressImplicitAnyIndexErrors": true,
         "strictNullChecks": false,
+        "lib": [
+            "dom", "es2015" // dom for console.log only
```

Comment:
We probably can use `es2015.core`.
Also, we can shun away from "dom" like here https://github.com/Microsoft/TypeScript/pull/13453/files